rpc/harmony/v1/types.go

```diff
@@ -346,6 +368,11 @@ func NewTxReceipt(
 		}
 	}
 
+	effectiveGasPrice := *big.NewInt(0)
+	if receipt.EffectiveGasPrice != nil {
```

Comment:
We should add an `else` clause to prevent the RPC to returns an effective gas price of 0. Not sure what edge case we are going to have in this case.